cmd/soroban-cli/Cargo.toml

```diff
@@ -71,6 +71,7 @@ jsonrpsee-http-client = "0.18.1"
 jsonrpsee-core = "0.18.1"
 hyper   = "0.14.27"
 hyper-tls = "0.5"
+openssl = { version = "0.10.55", features = ["vendored"] }
```

Comment:
@willemneal could you add the above line to your PR ? it would fix the aarch64 configuration build.
I'm still working on the windows issue.

Reply:
@tsachiherman Done! Also I removed the helper_url stuff from the config since we can use the RPC to get the URL